### ISSUE TITLE
Improve login window for small screens

### ISSUE THREAD

**Issue by guilhermednt** [redelivre/login-cidadao#34]
I'm working on improving the login page for small screen on PROCERGS#396. The proposed layout looks like this:

![home_login768](https://cloud.githubusercontent.com/assets/860796/15214706/f4aeeaf2-1824-11e6-910f-d123046bf729.png)

Are you interested in this change as well you should I do it only at PROCERGS' version?

**Comment by lpirola** [comment]
No, i bet this upgrade must be implemented in a way everyone could use. Anyone else?

**Comment by guilhermednt** [comment]
We have changes to this at PROCERGS#409, anyone interested?
